google-cloud-spanner/src/main/java/com/google/cloud/spanner/TransactionManager.java

```diff
@@ -61,6 +61,18 @@ enum TransactionState {
    */
   TransactionContext begin();
 
+  /**
+   * Initializes a new read-write transaction. This method must be called before performing any
```

Comment:
See my comment on `beginAsync`. I think that we should be more explicit in when this method should be used, and not describe it as 'useful in scenarios involving multiplexed sessions'. I think that could lead to wrong usage of this method.

Reply:
Done. Thanks for suggesting this